### DISCUSSION TITLE
Importing into day folders (as well as year/month)

### DISCUSSION THREAD

**Discussion by sunbeam60** [photoprism/photoprism#3357] [Q&A]
Hi,

A week ago I took a photo where, for the first time in my life, I perceived a massive difference between HDR in HEIC and SDR in jpeg. So I decided to move to HEIC as a my main storage format (both my wife and I take photos on iPhones).

To enable that I moved to uploading to the /import folder, rather than straight into /originals folder, then triggering an import rather than an index.

However, I now realise that PhotoPrism by default stores photos in year/month/ folder structure, whereas previously I uploaded into a year/month/day/ folder structure (since 1998, so I'm quite wedded to that format :)).

Is there a way to get PhotoPrims to import into year/month/day folders? It says on https://docs.photoprism.app/user-guide/library/import/ that one is free to use external batch renaming tools, but what does that mean? Can I move the original photos around after import without triggering a massive re-index? How, specifically, would one invoke such a batch renaming tool without upsetting PhotoPrism?

I should add that I self-host via docker, so if there's a docker-compose option for the folder structure instead of a GUI option, that's fine too.

**Comment by Issomane** [comment]
Hi, 

Have you found a solution to your issue? 
I have approximately the same issue but I want to go the other way around lol.
I just want the pictures in year folders.

Thanks!

**Comment by graciousgrey** [comment]
You can use a batch renaming tool or move files manually in your originals folder. In either case, reindexing is required afterwards so that PhotoPrism can pick up the changes.
